public/app/engagementindex/leaderboard/leaderboardController.js

```diff
@@ -334,6 +341,28 @@
       }
     };
 
+    /**
+     * Change daily notification settings for the course.
+     */
+    var changeDailyNotifications = $scope.changeDailyNotifications = function() {
+      if ($scope.me.is_admin) {
```

Comment:
@paulkerschen I see that `is_admin` is based on Canvas roles:
```
'ADMIN_ROLES': ['urn:lti:instrole:ims/lis/Administrator'],
'TEACHER_ROLES': ['ContentDeveloper', 'Instructor', 'urn:lti:role:ims/lis/TeachingAssistant'],
```
Are `ContentDeveloper` and `Instructor` custom (UCB) definitions or do they come out-of-the-box?